Source/Events/OnAuctionHouseClosed.lua

```diff
@@ -3,13 +3,19 @@ function Auctionator.Events.OnAuctionHouseClosed()
 
   AuctionatorAHFrame:Hide()
 
-  if Auctionator.FullScan.State.InProgress and not Auctionator.FullScan.State.Completed then
+  if Auctionator.FullScan.State.InProgress then
     Auctionator.FullScan.State.InProgress = false
-
-    Auctionator.Utilities.Message(
-      "Full scan failed to complete. " ..
-      Auctionator.FullScan.NextScanMessage()
-    )
+    if not Auctionator.FullScan.State.QuickCompleted then
+      Auctionator.Utilities.Message(
+        "Full scan failed to complete. " ..
+        Auctionator.FullScan.NextScanMessage()
+      )
+    elseif not Auctionator.DetailedCompleted then
```

Comment:
Should this be in the `Auctionator.FullScan.State` namespace?

Reply:
It should